### PR TITLE
Update manage-microsoft-365-groups-with-powershell.md

### PR DESCRIPTION
**Pull request by msbemba**
Corrected Add unified links desription

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3303

File: microsoft-365/enterprise/manage-microsoft-365-groups-with-powershell.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ The following cmdlets can be used with Microsoft 365 Groups.
 |[New-UnifiedGroup](https://go.microsoft.com/fwlink/p/?LinkId=616183) <br/> |Create a new Microsoft 365 Group. This cmdlet provides a minimal set of parameters. To set values for extended properties, use Set-UnifiedGroup after creating the new group  <br/> |
 |[Remove-UnifiedGroup](https://go.microsoft.com/fwlink/p/?LinkId=616186) <br/> |Delete an existing Microsoft 365 Group  <br/> |
 |[Get-UnifiedGroupLinks](https://go.microsoft.com/fwlink/p/?LinkId=616194) <br/> |Retrieve membership and owner information for a Microsoft 365 Group  <br/> |
-|[Add-UnifiedGroupLinks](https://go.microsoft.com/fwlink/p/?LinkId=616191) <br/> |Add hundreds or thousands of users, or new owners, to an existing Microsoft 365 Group  <br/> |
+|[Add-UnifiedGroupLinks](https://go.microsoft.com/fwlink/p/?LinkId=616191) <br/> |Add members, owners and subscribers to an existing Microsoft 365 Group <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/manage-microsoft-365-groups-with-powershell.md]:
```suggestion
|[Add-UnifiedGroupLinks](https://go.microsoft.com/fwlink/p/?LinkId=616191) <br/> |Add members, owners, and subscribers to an existing Microsoft 365 Group <br/> |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks.

**Pull request merged**